plugins/utils/navbuddy.nix

```diff
@@ -0,0 +1,275 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.navbuddy;
+  helpers = import ../helpers.nix {inherit lib;};
+  mkListStr = helpers.defaultNullOpts.mkNullable (types.listOf types.str);
+  percentageType = types.ints.between 0 100;
+  mkPercentageOpt = default: helpers.defaultNullOpts.mkNullable percentageType (toString default);
+in {
+  options.plugins.navbuddy =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "nvim-navbuddy";
+
+      package = helpers.mkPackageOption "nvim-navbuddy" pkgs.vimPlugins.nvim-navbuddy;
+
+      window = {
+        border = helpers.defaultNullOpts.mkBorder "rounded" "double" "solid" "none" ''
+          "rounded", "double", "solid", "none"  or an array with eight chars building up the border in a clockwise fashion
+          starting with the top-left corner. eg: { "╔", "═" ,"╗", "║", "╝", "═", "╚", "║" }.
+        '';
```

Comment:
`mkBorder` is wrongly called here and elsewhere as well.
It expects the `default` value for the option, the `name` of what the borders are for and an optional `desc`ription for this option (you can simply provide `""` if you don't want to provide it).

Function declaration: https://github.com/nix-community/nixvim/blob/05b77732e3babaa95d73cbffca83029784a64cdd/lib/helpers.nix#L140
Examples of how it is properly used: https://github.com/search?q=repo%3Anix-community%2Fnixvim%20mkBorder&type=code

Reply:
hahah yeh I just saw this error running the nix flake check locally. 

Ahh I see I though it was like an enum, not sure why! Fixing now.